qa/rpc-tests/getblocktemplate.py

```diff
@@ -49,11 +49,15 @@ def run_test(self):
 
         # Test 5: General checks
         tmpl = node.getblocktemplate()
-        assert(len(tmpl['noncerange']) == 16)
+        assert_equal(16, len(tmpl['noncerange']))
 
         # Test 6: coinbasetxn checks
         assert('foundersreward' in tmpl['coinbasetxn'])
         assert(tmpl['coinbasetxn']['required'])
 
+        # Test 7: hashFinalSaplingRoot checks
+        assert('finalsaplingroothash' in tmpl)
+        assert_equal(64, len(tmpl['finalsaplingroothash']))
```

Comment:
This should be the empty Sapling root, so we should be able to compare them directly.

Reply:
What do you mean by the empty Sapling root? If I print out ```tmpl['finalsaplingroothash']``` I get some seemingly random hash value (3e49b5f954aa9d3545bc6c37744661eea48d7c34e3000d82b7f0010c30f4c2fb).

Reply:
```
    uint256 expected = uint256S("3e49b5f954aa9d3545bc6c37744661eea48d7c34e3000d82b7f0010c30f4c2fb");

    ASSERT_TRUE(ZCSaplingIncrementalMerkleTree::empty_root() == expected);
```

I guess that's good!